server/app/grading_script.py

```diff
@@ -0,0 +1,56 @@
+import os
+import subprocess
+import json, urllib
+import httplib2
+import base64
+import ast
+import requests
+from oauth2client.tools import run
+from oauth2client.client import OAuth2WebServerFlow
+from oauth2client.file import Storage
+from apiclient.discovery import build
+from oauth2client.client import SignedJwtAssertionCredentials
+from oauth2client import gce
+
+http = httplib2.Http()
+email = '887076364996-opeevtug54963jn33liq618c1kjma8d5@developer.gserviceaccount.com'
```

Comment:
missing dependencies added to removed from enhancement/denero/autograde server_requirements.txt